pages/common/monodis.md

```diff
@@ -0,0 +1,31 @@
+# monodis
+
+> The Mono Common Intermediate Language (CIL) disassembler.
+
+- Disassemble an assembly to textual CIL:
```

Comment:
Should this be "Disassembly a binary" ? And same below where `exe`s are used.

Reply:
Indeed they are binaries, but `assembly` is the technical .NET term for a _Common Intermediate Language_ binary.